src/test/scala/org/broadinstitute/dsde/firecloud/dataaccess/MockRawlsDAO.scala

```diff
@@ -271,7 +273,7 @@ class MockRawlsDAO extends RawlsDAO {
     WorkspaceDetails(
       namespace = "namespace",
       name = "name",
-      workspaceId = "workspaceId",
+      workspaceId = mockWorkspaceId,
```

Comment:
Needed to switch this to a UUID to prevent an error attempting to initialize an actual `UUID` from the `workspaceId`